broker/applications/operators/src/docker-operator/DockerService.js

```diff
@@ -790,7 +790,8 @@ class DockerService extends BaseService {
       .logsAsync({
         stdout: 1,
         stderr: 1,
-        timestamps: 1
+        timestamps: 1,
+        follow: true
```

Comment:
To get the logs of the container, we need to pass the `follow` parameter as true which is `false` by default.

Reply:
Can you pls add a reference here as well/